src/injectJavaScriptWithWebViewKey.android.tsx

```diff
@@ -1,5 +1,5 @@
 import { NativeModules } from 'react-native';
 
-export default function injectJavaScriptWithWebViewKey(webViewKey: string, script: string) {
-  NativeModules.RNCWebView.injectJavaScriptWithWebViewKey(webViewKey, script);
+export default function injectJavaScriptWithWebViewKey(webViewKey: string, script: string): Promise<string | null> {
```

Comment:
in general the Promise generic type should should correspond to the non-error return-type of the function - in this case, I think Promise<void> is accurate

Reply:
Good call! I updated this to use `Promise<void>`